core/sdk-core/src/main/java/software/amazon/awssdk/core/exception/SdkServiceException.java

```diff
@@ -225,6 +231,17 @@ public Builder writableStackTrace(Boolean writableStackTrace) {
             return this;
         }
 
+        @Override
+        public Builder numAttempts(Integer numAttempts) {
+            super.numAttempts(numAttempts);
```

Comment:
Is there any reason we are calling super instead of this.numAttempts = numAttemps?